cypress/integration/functional/settings.spec.ts

```diff
@@ -9,7 +9,7 @@ context('wallet/settings', () => {
     cy.getByTestID('button_network_Playground').click()
   })
 
-  it('should exit wallet when clicked', function () {
+  it.only('should exit wallet when clicked', function () {
```

Comment:
only?

Reply:
Oh my bad haha i was testing it locally

Reply:
Hahaha, sorry I merged. I guess can enable again in another PR